### PR TITLE
Add role to attach an external drive and reconfigure HDFS to use it

### PR DESCRIPTION
**Pull request by OleksandrBerchenko**

### CODE REVIEW

**Comment by OleksandrBerchenko** [comment]
@Y4Rv1K, please take a look.

File: library/nova_volume.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
```
Comment (Y4Rv1K) [inline library/nova_volume.py]: I need context for that

**Comment by Y4Rv1K** [comment]
Looks fine. Testing required.

**Comment by dimmk0** [comment]
Testing is started.

**Pull request merged**